### PR TITLE
Raise image import limit

### PR DESCRIPTION
**Pull request by inflatador**
As of this week, the image import limit has been raised from 40GB to 160GB.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-082f1805ee/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-83767ccd49/how-to/). :bow:

File: content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ Images must follow these general requirements:
...
--   Images must not expand to a system disk larger than 40 GB.
+-   Images must not expand to a system disk larger than 160 GB.
 
-    **Note**: Images with system disks larger than 40 GB can be exported but
+    **Note**: Images with system disks larger than 160 GB can be exported but
     cannot be imported into the Rackspace Open Cloud.
 
--   If you have exported an image from the Rackspace Open Cloud it will
+-   If you have exported an image from the Rackspace Open Cloud it is
     already be in the VHD format that is required for import.
```
Comment (cyrichardson) [inline content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md]: already in (delete be)

File: content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md [hunk 2]
```diff
@@ -26,12 +26,12 @@ Images must follow these general requirements:
...
 
--   Images must not expand to a system disk larger than 40 GB.
+-   Images must not expand to a system disk larger than 160 GB.
 
-    **Note**: Images with system disks larger than 40 GB can be exported but
+    **Note**: Images with system disks larger than 160 GB can be exported but
     cannot be imported into the Rackspace Open Cloud.
 
--   If you have exported an image from the Rackspace Open Cloud it will
+-   If you have exported an image from the Rackspace Open Cloud it is
```
Comment (cyrichardson) [inline content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md]: Cloud, it

File: content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md (hunk 1 above)
Comment (catlook) [reply content/cloud-images/preparing-an-image-for-import-into-the-rackspace-opencloud.md]: lol

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fef26bde70/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b409cb8433/how-to/)

**Pull request merged**